app/controllers/middleware_server_controller.rb

```diff
@@ -74,12 +73,12 @@ def show
 
     if @display == 'middleware_datasources'
       @gtl_url = '/show'
-      show_container_display(@record, 'middleware_datasource', MiddlewareDatasource)
+      show_middleware_display(@record, 'middleware_datasource', MiddlewareDatasource)
     elsif @display == 'middleware_deployments'
       @gtl_url = '/show'
```

Comment:
why isn't this set inside `show_middleware_display` ? (similar to `show_middleware`)